<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>554</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>554</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mcmap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="renderButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="saveSelect">
       <item>
        <widget class="QLabel" name="saveFieldLabel">
         <property name="text">
          <string>Savefile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="saveNameLabel">
         <property name="text">
          <string>__Not selected.__</string>
         </property>
         <property name="textFormat">
          <enum>Qt::MarkdownText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveSelectButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="outputSelect">
       <item>
        <widget class="QLabel" name="outputFieldLabel">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>output.png</string>
         </property>
         <property name="textFormat">
          <enum>Qt::MarkdownText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="outputSelectButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="dimensionSelect">
       <item>
        <widget class="QLabel" name="dimensionLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Dimension</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dimensionSelectDropDown">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="xBounds">
       <item>
        <widget class="QLabel" name="minXLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>From X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minX">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maxXLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>To X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxX">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="zBounds">
       <item>
        <widget class="QLabel" name="minZLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>From Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minZ">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maxZLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>To Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxZ">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="yBounds">
       <item>
        <widget class="QLabel" name="minYLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>From Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minY">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maxYLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>To Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxY">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="paddingFieldLabel">
         <property name="text">
          <string>Image Padding</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="paddingValue">
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="shading">
       <property name="text">
        <string>Shading</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="misc">
       <item>
        <widget class="QCheckBox" name="hideBeacons">
         <property name="text">
          <string>Hide Beacons</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hideWater">
         <property name="text">
          <string>Hide Water</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="colorSelect">
       <item>
        <widget class="QLabel" name="colorFieldLabel">
         <property name="text">
          <string>Colorfile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colorFileLabel">
         <property name="text">
          <string>Not selected.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorSelectButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumcmap">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menumcmap"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
